client/src/store/index.ts

```diff
@@ -1,4 +1,5 @@
-import Promise from 'bluebird';
+import BluebirdPromise from 'bluebird';
+import { createDirectStore } from "direct-vuex"
```

Comment:
Does our linter enforce quote characters and semicolons on these?

Reply:
It does, I just forgot to run it :/